UnityProject/Assets/Scripts/Construction/ConveyorBelts/ConveyorBelt.cs

```diff
@@ -317,4 +317,18 @@ public void ServerPerformInteraction(HandApply interaction)
 				});
 		}
 	}
+
+	//######################################## Multitool interaction ##################################
```

Comment:
Might as well make this a `Header["Multitool interaction"]`

Reply:
How does it work exactly?

Reply:
Oh, headers are just text that appears in the inspector, like a headline. 

So you'll see that the following serialized variables are for "Multitool interaction", but it'll also be visible in the inspector 🙂

https://docs.unity3d.com/ScriptReference/HeaderAttribute.html

Reply:
oh ok, thought it was some type of code separation stuff